app/controllers/users/notes/ratings_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Users
+  module Notes
+    class RatingsController < ApplicationController
+      expose :user, find_by: :username # author
+      expose :note, find_by: :slug, parent: :user
+      expose :ratings, from: :note
+      expose :rating, parent: :note
+
+      respond_to :json
+
+      def create
+        ::Notes::Ratings::Create.call(rating: rating, user: current_user)
+
+        respond_with user, note, rating
+      end
+
+      def update
+        rating.assign_attributes(rating_params)
+        ::Notes::Ratings::Update.call(rating: rating)
+
+        respond_with rating
+      end
+
+      def destroy
+        ::Notes::Ratings::Destroy.call(rating: rating)
+
+        respond_with user, note, rating
+      end
+
+      private
+
+      def rating_params
+        params.require(:rating).permit(:note_id, :value)
```

Comment:
```suggestion
        params.require(:rating).permit(:value)
```

`note_id` is a query string parameter, you shouldn't permit it here.